src/test/java/uk/gov/hmcts/cmc/claimstore/services/ClaimServiceTest.java

```diff
@@ -209,6 +210,28 @@ public void saveClaimShouldFinishSuccessfully() {
         verify(ccdEventProducer, once()).createCCDClaimIssuedEvent(eq(createdClaim), eq(USER));
     }
 
+    @Test
+    public void saveLegalRepClaimShouldFinishSuccessfully() {
+
+        ClaimData claimData = SampleClaimData.validDefaults();
+
+        when(userService.getUser(eq(AUTHORISATION))).thenReturn(USER);
+        when(caseRepository.saveClaim(eq(USER), any())).thenReturn(claim);
+
+        Claim createdLegalRepClaim = claimService
+            .saveLegalRepClaim(
+                USER_ID, claimData, AUTHORISATION);
+
+        assertThat(createdLegalRepClaim.getClaimData()).isEqualTo(claim.getClaimData());
+
+        verify(caseRepository, once()).saveClaim(
```

Comment:
no need to specify `once()`

Reply:
i now understand that once() is default because of your comment, thanks :+1:.

I think i prefer to be explicit and also same as saveClaim test.